src/main/java/com/robosh/ejournal/entity/Mark.java

```diff
@@ -0,0 +1,28 @@
+package com.robosh.ejournal.entity;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
```

Comment:
remove this import with wildcard. Need to be discussed with team about code style: with what number imports we need to use wildcard imports.